src/server/config/transform_deprecations.js

```diff
@@ -25,6 +25,18 @@ const savedObjectsIndexCheckTimeout = (settings, log) => {
   }
 };
 
+const rewriteBasePath = (settings, log) => {
+  if (_.has(settings, 'server.basePath') && !_.has(settings, 'server.rewriteBasePath')) {
+    log(
```

Comment:
__question:__ is there any magic step I should make to see this warning? I don't see any deprecation warnings defined in this file for some reason :/

Reply:
Wow, well, looks like the server isn't logging deprecation messages... https://github.com/elastic/kibana/issues/16795. I think you'll just have to verify the code until @epixa gets to https://github.com/elastic/kibana/issues/16795

Reply:
Haha, yeah code lgtm